Doc/library/urllib.request.rst

```diff
@@ -725,11 +727,11 @@ The following attribute and methods should only be used by classes derived from
    This method will be called before any protocol-specific open method.
 
 
-.. method:: BaseHandler.protocol_open(req)
+.. method:: BaseHandler.<protocol>_open(req)
    :noindex:
 
    This method is *not* defined in :class:`BaseHandler`, but subclasses should
-   define it if they want to handle URLs with the given protocol.
+   define it if they want to handle URLs with the given *protocol*.
```

Comment:
I think the originals make more sense; _protocol_ is used as a word in these cases.

Reply:
I interpreted it to mean that `protocol` was the actual "parameter" which was passed in, rather than protocol in the sense of a word.

Reply:
I think that should be <protocol> because you are talking about *the* protocol, isn't?